_meta/up_districts.meta

```diff
@@ -1,196 +1,210 @@
-आगरा  ,  Agra
-अलीगढ  ,  Aligarh
-अलीगढ़ , Aligarh
-अलीगढ़ , Aligarh
-अलीगढ़ , Aligarh
-अमेठी  ,  Amethi
-अमेठी  ,  Amethi
-अम्बेडकरनगर, Ambedkar Nagar
-अम्बेडकर, Ambedkar Nagar
-अम्बेडकर नगर , Ambedkar Nagar
-अमरोहा , Amroha
-अमरोह , Amroha
-औरय्या , Auraiya
-औरैय्या  ,Auraiya
-औरैय्य , Auraiya
-अयोध्या  , Ayodhya
-अयोध्य , Ayodhya
-आजमगढ़  ,  Azamgarh
-आजमगढ , Azamgarh
-आजमगढ़ , Azamgarh
-बहराइच , Bahraich
-बहर इच , Bahraich
-बनिया , Ballia
-बलिया  ,  Ballia
-बललया , Ballia
-बमलया , Ballia
+  आगरा  ,  Agra
```

Comment:
This should ideally be a separate Pull request because I can't seem to approve this comment individually.